config/rector/set/laminas-servicemanager-40.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Renaming\Rector\Name\RenameClassRector;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->set(RenameClassRector::class)
+        ->call('configure', [[
+        RenameClassRector::OLD_TO_NEW_CLASSES => [
+            // interop deprecated interop to psr
+            'Interop\Container\ContainerInterface' => 'Psr\Container\ContainerInterface',
+
+            // update deprecated root factory to new factory under new namespace
+            'Laminas\ServiceManager\AbstractFactoryInterface' => 'Laminas\ServiceManager\Factory\AbstractFactoryInterface',
+            'Laminas\ServiceManager\FactoryInterface' => 'Laminas\ServiceManager\Factory\FactoryInterface',
+            'Laminas\ServiceManager\DelegatorFactoryInterface' => 'Laminas\ServiceManager\Factory\DelegatorFactoryInterface',
+            'Laminas\ServiceManager\InitializerInterface' => 'Laminas\ServiceManager\Initializer\InitializerInterface',
+
+            // update deprecated root factory to new factory under new namespace for laminas/laminas-zendframework-bridge
+            'Zend\ServiceManager\AbstractFactoryInterface' => 'Laminas\ServiceManager\Factory\AbstractFactoryInterface',
```

Comment:
Hm, I would not do that. We offer `laminas-migration` which helps migrating a package/project to migrate to laminas. No need to rename interfaces like this imho.

With `laminas-servicemanager` v4, we are removing the zendframework bridge along with the `replace` statement (as there is no v4 in zend).

If any user is still requiring `zend-servicemanager`, we should point them to the migration before updating to `laminas-servicemanager` v4.

Reply:
removed.